pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1091,18 +1061,36 @@ func (n *NetworkPolicyController) syncAddressGroup(key string) error {
 		utilsets.MergeString(addrGroupNodeNames, internalNP.SpanMeta.NodeNames)
 	}
 	memberSet := n.getAddressGroupMemberSet(addressGroup)
+	if addressGroup.Selector.NodeSelector != nil {
+		ms, nodes := n.addNodeSelectorMemberSet(addressGroup.Selector.NodeSelector)
+		memberSet = memberSet.Union(ms)
+		utilsets.MergeString(addrGroupNodeNames, nodes)
+	}
 	updatedAddressGroup := &antreatypes.AddressGroup{
 		Name:         addressGroup.Name,
 		UID:          addressGroup.UID,
 		Selector:     addressGroup.Selector,
 		GroupMembers: memberSet,
 		SpanMeta:     antreatypes.SpanMeta{NodeNames: addrGroupNodeNames},
 	}
+	klog.InfoS("Sync address group", "key", key, "updateTo", updatedAddressGroup)
 	klog.V(2).Infof("Updating existing AddressGroup %s with %d Pods/ExternalEntities and %d Nodes", key, len(memberSet), addrGroupNodeNames.Len())
 	n.addressGroupStore.Update(updatedAddressGroup)
 	return nil
 }
 
+func (c *NetworkPolicyController) addNodeSelectorMemberSet(selector labels.Selector) (controlplane.GroupMemberSet, sets.String) {
+	groupMemberSet := controlplane.GroupMemberSet{}
+	nodes, _ := c.nodeLister.List(selector)
+	nList := sets.String{}
+	for _, node := range nodes {
+		groupMemberSet.Insert(nodeToGroupMember(node))
+		nList.Insert(node.Name)
+	}
+	klog.InfoS("Select node", "nList", nList, "groupMemberSet", groupMemberSet, "selector", selector)
```

Comment:
Such verbose log with all member informtion is not suitable for V(0). And we should avoid dumping all members which may be thousands level in any log level.

Reply:
Done.